plugins/processors/defaulter/README.md

```diff
@@ -0,0 +1,43 @@
+# Defaulter Processor
```

Comment:
Let's call the plugin either `defaults` or `default`, this fits the noun style of most of our processors.

Reply:
Gotcha, `defaults` is a great replacement!